edge/src/service/github/github.service.ts

```diff
@@ -32,6 +34,40 @@ class GithubService extends HttpClient {
       path: `/orgs/${this.organizationName}/members`,
     });
   }
+
+  async getOrgRepos(): Promise<[GithubRepositoryModel]> {
+    return await this.call<[GithubRepositoryModel]>({
+      method: HttpMethod.GET,
+      path: `/orgs/${this.organizationName}/repos`,
+    });
+  }
+
+  async getOrgContributors(): Promise<GithubContributorModel[]> {
+    const repos = await this.getOrgRepos();
+    const promises = repos.map((repo) => {
+      return this.call<GithubContributorModel[]>({
+        method: HttpMethod.GET,
+        path: `/repos/${this.organizationName}/${repo.name}/contributors`,
+      });
+    });
+
+    return Promise.all(promises).then((repoContributions) => {
+      const aggregation: Record<string, GithubContributorModel> = {};
+
+      repoContributions.forEach((contributors) => {
+        contributors.forEach((contributor) => {
+          if (!aggregation[contributor.id]) {
```

Comment:
IMHO, we can use object.hasOwnProperty(propName) for validation in here

Reply:
ah good idea, will update thanks